### PR TITLE
Revert sudo/update python deb

### PR DESCRIPTION
**Pull request by kzidane**
cc @dmalan @glennholloway

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -38,7 +37,7 @@ deb: clean Makefile
 	echo "version=$(VERSION)" > "$(VERSION_FILE)"
 
 	# set permissions
-	chmod -R 755 "$(USR_DIR)/bin" "$(USR_DIR)/local/bin"
+	chmod -R 755 "$(FILES_DIR)/usr/bin/"
```
Comment (dmalan) [inline Makefile]: Didn't we decide it's better to `chmod` in `post-install.sh`?

Comment (kzidane) [inline Makefile]: Yes, but then we decided to use `fpm` to our advantage. See https://github.com/cs50/ide50/pull/30!

**Comment by dmalan** [comment]
Hm, didn't we run into some case where permissions were not somehow
preserved? Should we ever be setting permissions in post-install.sh?

**Comment by kzidane** [comment]
Just searched through the emails, and it seems @glennholloway had permission problems with all our binaries, along with `.prompt50`. It was the time around the sudoers problem, so that might have been the reason, especially that the pattern of the permissions was `*00`, which makes sense, because the umask then was `0077`?

I couldn't replicate, at that time, though, so not entirely sure that was the reason!

**Comment by kzidane** [comment]
I suppose, normally, permissions are set in postinst, but it's quite a pain to hard-code the files and directories to be chmoded, and keep track of which was added/renamed/removed, etc, especially, when we can make use of that (since it's confirmed by an author)?

**Comment by dmalan** [comment]
Not opposed, but we're talking about having pretty much the same `chmod` lines in `Makefile` instead of `post-install.sh`, right?

**Comment by kzidane** [comment]
Yes, same `chmod`s, except we're able to do some stuff recursively, without touching the files of other packages, benefit from the variables, we have defined, in the `Makefile`, and if some file is missing, we catch that at build time.

**Comment by dmalan** [comment]
Ah, makes sense.

**Comment by kzidane** [comment]
[Merged](https://github.com/cs50/ide50/pull/33/commits/3cbec1a1eb42c9924c945f4666d967f03fb2595f) into configs!

**Pull request closed without merging**